README.md

```diff
@@ -11,7 +11,7 @@ repositories {
 }
 
 dependencies {
-    compile 'com.beust:klaxon:0.29'
```

Comment:
Can you just update the version and leave the line as is? Less verbose this way.